webapp/src/components/MarketplacePage/Publication/Publication.js

```diff
@@ -24,6 +24,34 @@ export default class Publication extends React.PureComponent {
   static defaultProps = {
     isOwnerVisible: true
   }
+
+  renderButton({ publication, isExpired }) {
+    let text = 'View'
+    if (publication.status === PUBLICATION_STATUS.sold) {
+      text = 'Sold'
+    }
+    if (publication.status === PUBLICATION_STATUS.canceled) {
```

Comment:
Hudston we have a problem. The server uses `cancelled` and the front `canceled`. Don't know how to avoid this without sharing code between the two.

I think this PR is a good place to fix it, I think `canceled` looks nicer, but the Marketplace contract uses `AuctionCancelled` so either choice is fine.